### PR TITLE
add multiapiclient

### PR DESCRIPTION
**Pull request by xiangyan99**

### CODE REVIEW

**Comment by xiangyan99** [comment]
#10179

File: sdk/core/azure-core/azure/core/profiles/multiapi_client_mixin.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
+#-------------------------------------------------------------------------
```
Comment (lmazuel) [inline sdk/core/azure-core/azure/core/profiles/multiapi_client_mixin.py:1]: file should be _ prefixed and this class accessible from azure.core.profiles

**Comment by xiangyan99** [comment]
Dup of ##17284

**Pull request closed without merging**